### ISSUE TITLE
JOSS 3: Software intent and comparison to other software

### ISSUE THREAD

**Issue by dlagrava** [roshansamuel/saras#3]
Hello,

I have been reviewing your software. As part of the checks that I need to perform, I need to understand what this software has been written for ie. do you have a particular paper that needs it, because other software do not work? Why did you write the software?

Also, it would be good to explain and discuss why you chose to go with something like finite differences, which are out of favor on most popular CFD solvers.

Cheers,
Daniel

**Comment by roshansamuel** [comment]
Hi,

Thank you very much for the helpful review comments. In the commit ad5735b, the [Summary](https://github.com/roshansamuel/saras/blob/master/paper.md#summary) section of the paper.md file has been expanded. We have added a brief explanation on intent of software. We also cite a recent paper which uses results from the SARAS code. 

Although many popular CFD solvers now use finite-volume method or newer methods like LBM, finite-difference methods still offer comparatively higher accuracy on regular Cartesian grids. We wrote SARAS with this emphasis on accuracy and speed for academic problems.

If you feel that the expanded summary of the paper sufficiently explains the intent of the software, kindly mark this issue as closed.

Thanks!

Roshan
